packages/ui/src/Select.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import * as React from 'react';
+
+/**
+ * The class name added to all UI components.
+ */
+const UI_CLASS = 'jp-ui';
+
+/**
+ * The class name added to wrapper div.
+ */
+const WRAPPER_CLASS = 'jp-select-wrapper';
+
+/**
+ * The class name added to styled select.
+ */
+const STYLED_CLASS = 'jp-mod-styled';
+
+/**
+ * The class name added to a focused select element.
+ */
+const SELECT_FOCUSED_CLASS = 'jp-mod-focused';
+
+/**
+ * The properties of the Select component.
+ */
+export interface ISelectProps {
+  children?: any;
+  className?: string;
+  onChange?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
```

Comment:
What do you think about changing this callback to just take a `string` which is the updated value? That way the user doesn't need to know about react change events?